framework/src/media/utils/MediaUtils.cpp

```diff
@@ -617,18 +619,42 @@ static bool isMpeg2Ts(const unsigned char *buffer, size_t size)
 	return true;
 }
 
-static bool isMp3(const unsigned char *buffer, size_t size)
+static bool checkId3V2Header(const unsigned char *buffer, size_t size, bool isFile, FILE *fp = nullptr)
```

Comment:
only fp pointer is enough, if fp is NULL lets use buffer else consider file

Reply:
Changes done